sui/its.js

```diff
@@ -228,15 +229,35 @@ async function registerCustomCoin(keypair, client, config, contracts, args, opti
     const deployConfig = { client, keypair, options, walletAddress };
     const [symbol, name, decimals] = args;
 
+    const unvalidatedParams = { isNonEmptyString: { symbol, name } };
```

Comment:
```suggestion
    validateParameters({ isNonEmptyString: { symbol, name } });
```
a bit confusing, just call `validateParameters` twice for clarity